lhotse/dataset/speech_recognition.py

```diff
@@ -8,6 +8,9 @@
 from lhotse.cut import CutSet
 from lhotse.utils import Pathlike
 
+# TODO: make them optional
+import simplefst
+from pychain import ChainGraph, ChainGraphBatch
```

Comment:
the easiest way to make this optional is to do a local import (i.e. import in a function with a very limited scope)